app/services/rbac_api.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# This class is meant to handle calls to the RBAC light API
+class RBACAPI
+  def initialize(account)
+    @url = URI.parse("#{URI.parse(Settings.remediations_url)}"\
+                     "#{ENV['PATH_PREFIX']}/rbac/v1/access")
+    @b64_identity = Base64.strict_encode64(account.fake_identity_header.to_json)
+  end
+
+  def check_user
+    body = JSON.parse(access_check_response)
+    body['data'].find do |access_check|
+      return true if access_check['permission'] == 'compliance:*:*'
```

Comment:
For now, that's the only check we have to make - https://github.com/RedHatInsights/rbac-config/blob/master/configs/compliance.json